### DISCUSSION TITLE
Transfer Folders size issues

### DISCUSSION THREAD

**Discussion by Smiley006** [iterate-ch/cyberduck#16354] [Q&A]
Hello, I'm trying to set up a new transfer and I do what I've always done, except the folder I'm trying to upload is said to be 50 GB in size, while the actual original folder is only 11 GB size. The step I take are: I go to my bookmark, select the service, identify the Location for the destination upload, click action, select upload, identify the source upload folder, click upload. This is the process I've always used that has always worked until now. Running cyberduck 9.0.2 on an Intel Mac running Sonoma 14.6.1. Any help you could provide will be greatly appreciated. Thanks very much, take care, Harry

**Comment by tatze96** [comment]
Is the described behavior still present?

If so, can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-16354 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Reply by Smiley006** [reply]
Hello and thanks for your email. I was copying the files originally into an empty folder and perhaps that had something to do with it. I then uploaded the files directly to the webpage of the Microsoft account. After that, I tried again and sure enough, it only copied the new files, which were approximately 3.2 GB in size. The last time I tried uploading to an empty folder, it said that 2 TB of files were to be transferred. I'm not sure exactly why directly uploading the files to the website made such a difference, but at this point it does seem to be copying the appropriate volume of files.
